receiver/bms_stream_formatter.py

```diff
@@ -0,0 +1,24 @@
+  
+import sys
+import ast
+import pandas as pd
+
+def bms_format_console_pipeline():
```

Comment:
Then name `bms_format_console_pipeline` is not very clear to me... think about what you want readers to understand when they see the function name.